File: implementation/time/time.py
# 시간 입력 받기
h = int(input())

count = 0 # 처음 카운트 초기화
for i in range(h + 1): # 시간
    for j in range(60): # 분
        for k in range(60): # 초
            if "3" in str(i) + str(j) + str(k): # 3이 시간이나 분 초에 들어가면
                count += 1 # 카운트 1 더하기

print(count)

File: implementation/time/time.java
package implementation.time;

import java.util.*;

public class time {
    public static boolean check(int h, int m, int s) {
        if (h % 10 == 3 || m / 10 == 3 || m % 10 == 3 || s / 10 == 3 || s % 10 == 3)
            return true;
        return false;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int h = sc.nextInt();
        int count = 0;

        for (int i = 0; i < h + 1; i++) {
            for (int j = 0; j < 60; j++) {
                for (int k = 0; k < 60; k++) {
                    if (check(i, j, k)) {
                        count++;
                    }
                }
            }
        }
        System.out.println(count);
    }
}